src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,22 @@
 <html>
 	<head>
 		<script>
+
+			const onVsCodeWorkbenchError = (event) => {
+				const flags = new Map();
+
+				// We take a look at what is the resource that was attempted to load;
+				const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
+
+				// If the event has a `target`, it means that it wasn't a script error
+				if (resourceSource) {
+					flags.set('resource', 'vscode-web-workbench');
+					flags.set('error', 'load_error');
+				}
+
+				window?.gitpodMetricsSupervisorFrontendErrorTotal(flags)
```

Comment:
@mustard-mh no-one can guarantee that supervisor frontend can be loaded, we should use here plain browser fetch api, it is the point of having it